### PR TITLE
Bump package dep on ansi-regex to unblock dependent chalk package.

### PR DESCRIPTION
**Pull request by ChrisRus**
Clone of current master branch and attempt to npm install fails on ansi-regex dependency requiring minor package adjustment. Thanks.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "strip-ansi",
-  "version": "2.0.1",
+  "version": "2.0.2",
```
Comment (jbnicolai) [inline package.json]: Please don't bump the package version. The maintainers will do that when they release a new version.

File: package.json [hunk 2]
```diff
@@ -48,7 +48,7 @@
     "text"
   ],
   "dependencies": {
-    "ansi-regex": "^1.0.0"
+    "ansi-regex": "~1.1.0"
```
Comment (jbnicolai) [inline package.json]: Why did you replace the `^` for a `~`?

**Comment by ChrisRus** [comment]
Hi. Just a quick job here w/npm overrides. I can cancel pull if you would prefer no. Actually just trying to get get grunt-contrib-coffee package install unblocked and following the break chain down.

**Comment by jbnicolai** [comment]
I've fixed this on master in cee0461 and release a new patch `v2.0.2`.

**Comment by ChrisRus** [comment]
thanks

**Pull request closed without merging**